### DISCUSSION TITLE
Transitioning to Asset-Lite Meterless Payment and Sign Fabrication

### DISCUSSION THREAD

**Discussion by StephenHDDOT** [openmobilityfoundation/curb-data-specification#118] [Policy]
Hello Curb OMF group members,

I am writing from DDOT, the District Department of Transportation in Washington DC.  Our curbside management division is in the process of transitioning to asset lite meterless payments and are having some questions regarding how other municipalities manage their mobile payment options.  Specifically questions such as if other cities have multiple vendors for parking pay-by-cell options, are signs fabricated and installed in-house by cities or printed and installed by the vendor?  

While not specifically a CDS question, some of our transitioning to asset lite, meterless and mobile payment has potential integration with CDS and a more digital way of managing the curb.  We are having some internal discussions and challenges with deciding on who should be responsible for zone # fabrication on the sign, whether our city should manage that or if the vendor should manage that and what would process would be best for asset management.  Wanted to ask this group as a repository of curb thinkers regarding what their municipal policies are regarding asset lite/ mobile payments and what their directions are for that.

Happy Friday!

**Comment by jlarsonOmahaNE** [comment]
Hey @StephenHDDOT We are implementing our [new system](https://www.parkomaha.com/news/announcements/99-park-omaha-lowers-initial-rates-and-eliminates-meter-time-limits) and did all of the rezoning and signs/stickers for both on and off street ourselves. Feel free to reach out via email!
